lib/cnab/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cnab
-  VERSION = "0.4.3"
+  VERSION = "0.4.4"
```

Comment:
@eduardoazevedo3 Could you please remove this change?

Meanwhile i'm gonna try to get access to this repository again. I only have access to the gem right now.